deepfence_worker/cronjobs/threat.go

```diff
@@ -2,14 +2,17 @@ package cronjobs
 
 import (
 	"sync/atomic"
+	"time"
 
 	"github.com/ThreeDotsLabs/watermill/message"
 
 	"github.com/deepfence/golang_deepfence_sdk/utils/directory"
+	"github.com/deepfence/golang_deepfence_sdk/utils/log"
 	"github.com/neo4j/neo4j-go-driver/v4/neo4j"
 )
 
 var threatGraphRunning atomic.Bool
+var exploitabilityRunning atomic.Bool
```

Comment:
I think these flags needs to be set and checked from redis as the jobs might be running on different workers

Reply:
Good point, more globally I am not sure worker is actually a scalable process
For one, it has all the cron jobs, which should all be unique

Reply:
Scheduler is unique and triggers cron jobs. Worker should be scalable. @noboruma